src/components/StarRating/StarRatingItem.tsx

```diff
@@ -0,0 +1,28 @@
+interface StarRatingItemProps {
+  initFill: boolean
+  handleChangeStar: () => void
+}
+
+const StarRatingItem = ({
+  initFill,
+  handleChangeStar,
+}: StarRatingItemProps) => {
+  return (
+    <svg
+      width="36"
+      height="35"
+      viewBox="0 0 36 35"
+      className="inline cursor-pointer"
+      xmlns="http://www.w3.org/2000/svg"
+      onClick={handleChangeStar}
+      onMouseEnter={handleChangeStar}
+    >
+      <path
+        d="M27.3601 21.1559C27.1734 21.3199 27.0345 21.5315 26.9584 21.7681C26.8822 22.0047 26.8715 22.2575 26.9275 22.4997L29.2302 32.47C29.291 32.7304 29.2739 33.003 29.181 33.2537C29.088 33.5045 28.9233 33.7223 28.7074 33.8801C28.4915 34.0379 28.2339 34.1287 27.9668 34.1412C27.6997 34.1536 27.4348 34.0872 27.2051 33.9501L18.5019 28.6703C18.2904 28.5417 18.0477 28.4737 17.8002 28.4737C17.5527 28.4737 17.31 28.5417 17.0985 28.6703L8.39528 33.9501C8.16565 34.0872 7.90076 34.1536 7.63362 34.1412C7.36649 34.1287 7.10892 34.0379 6.89301 33.8801C6.67711 33.7223 6.51241 33.5045 6.41945 33.2537C6.32648 33.003 6.30936 32.7304 6.37021 32.47L8.6729 22.4997C8.72887 22.2575 8.71821 22.0047 8.64204 21.7681C8.56587 21.5315 8.42704 21.3199 8.24029 21.1559L0.557282 14.4539C0.351879 14.2793 0.202876 14.0477 0.129128 13.7884C0.0553806 13.5291 0.060202 13.2538 0.142982 12.9973C0.225762 12.7407 0.382781 12.5145 0.59417 12.3472C0.805559 12.1799 1.06182 12.0791 1.33052 12.0575L11.4576 11.1821C11.7047 11.1601 11.941 11.071 12.1412 10.9246C12.3414 10.7781 12.4978 10.5798 12.5936 10.3509L16.5501 0.92217C16.6558 0.678399 16.8304 0.470841 17.0525 0.325042C17.2746 0.179242 17.5345 0.101562 17.8002 0.101562C18.0659 0.101562 18.3258 0.179242 18.5479 0.325042C18.77 0.470841 18.9447 0.678399 19.0503 0.92217L23.0068 10.3509C23.1026 10.5798 23.259 10.7781 23.4592 10.9246C23.6594 11.071 23.8957 11.1601 24.1428 11.1821L34.2699 12.0575C34.5386 12.0791 34.7948 12.1799 35.0062 12.3472C35.2176 12.5145 35.3746 12.7407 35.4574 12.9973C35.5402 13.2538 35.545 13.5291 35.4713 13.7884C35.3975 14.0477 35.2485 14.2793 35.0431 14.4539L27.3601 21.1559Z"
+        fill={initFill ? '#FFC700' : 'black'}
+      />
+    </svg>
```

Comment:
이 부분은 아이콘 컴포넌트 만들고, 직접 넣으면 더 간편해요!

```suggestion
    <StarIcon
      fill={initFill ? '#FFC700' : 'black'}
      onClick={handleChangeStar}
      onMouseEnter={handleChangeStar}
    />
```

Reply:
```svg
<svg
  width="36"
  height="35"
  viewBox="0 0 36 35"
  className="inline cursor-pointer"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M27.3601 21.1559C27.1734 21.3199 27.0345 21.5315 26.9584 21.7681C26.8822 22.0047 26.8715 22.2575 26.9275 22.4997L29.2302 32.47C29.291 32.7304 29.2739 33.003 29.181 33.2537C29.088 33.5045 28.9233 33.7223 28.7074 33.8801C28.4915 34.0379 28.2339 34.1287 27.9668 34.1412C27.6997 34.1536 27.4348 34.0872 27.2051 33.9501L18.5019 28.6703C18.2904 28.5417 18.0477 28.4737 17.8002 28.4737C17.5527 28.4737 17.31 28.5417 17.0985 28.6703L8.39528 33.9501C8.16565 34.0872 7.90076 34.1536 7.63362 34.1412C7.36649 34.1287 7.10892 34.0379 6.89301 33.8801C6.67711 33.7223 6.51241 33.5045 6.41945 33.2537C6.32648 33.003 6.30936 32.7304 6.37021 32.47L8.6729 22.4997C8.72887 22.2575 8.71821 22.0047 8.64204 21.7681C8.56587 21.5315 8.42704 21.3199 8.24029 21.1559L0.557282 14.4539C0.351879 14.2793 0.202876 14.0477 0.129128 13.7884C0.0553806 13.5291 0.060202 13.2538 0.142982 12.9973C0.225762 12.7407 0.382781 12.5145 0.59417 12.3472C0.805559 12.1799 1.06182 12.0791 1.33052 12.0575L11.4576 11.1821C11.7047 11.1601 11.941 11.071 12.1412 10.9246C12.3414 10.7781 12.4978 10.5798 12.5936 10.3509L16.5501 0.92217C16.6558 0.678399 16.8304 0.470841 17.0525 0.325042C17.2746 0.179242 17.5345 0.101562 17.8002 0.101562C18.0659 0.101562 18.3258 0.179242 18.5479 0.325042C18.77 0.470841 18.9447 0.678399 19.0503 0.92217L23.0068 10.3509C23.1026 10.5798 23.259 10.7781 23.4592 10.9246C23.6594 11.071 23.8957 11.1601 24.1428 11.1821L34.2699 12.0575C34.5386 12.0791 34.7948 12.1799 35.0062 12.3472C35.2176 12.5145 35.3746 12.7407 35.4574 12.9973C35.5402 13.2538 35.545 13.5291 35.4713 13.7884C35.3975 14.0477 35.2485 14.2793 35.0431 14.4539L27.3601 21.1559Z"
  />
</svg>
```

```ts
export { default as StarIcon } from './star.svg?react'
```

Reply:
![q](https://github.com/prgrms-web-devcourse/Team-12-ReviewRanger-FE/assets/97094709/a38b411e-852b-4a02-bf69-c9b1485bbcd8)
잘 적용이 됩니다!!